Tasks/CopyFilesToMachines/CopyFilesToMachines.ps1

```diff
@@ -21,13 +21,13 @@ Write-Verbose "cleanTargetBeforeCopy = $cleanTargetBeforeCopy" -Verbose
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Internal"
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs"
 
+[System.Reflection.Assembly]::LoadFrom("$env:AGENT_HOMEDIRECTORY\Agent\Worker\Modules\Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs\Microsoft.VisualStudio.Services.DevTestLabs.Common.dll")
```

Comment:
Would it be better to do this from the helper where you really load and consume the dll?